modules/rosa-hcp-firewall-prerequisites.adoc

```diff
@@ -106,30 +90,29 @@ endif::rosa-classic-sts[]
 Managed clusters require enabling telemetry to allow Red{nbsp}Hat to react more quickly to problems, better support the customers, and better understand how product upgrades impact clusters.
```

Comment:
This Plus sign shows up in the live build.

![image](https://github.com/user-attachments/assets/06ce8c8b-dba4-490d-91f7-7b7285cfbdf4)

Reply:
Think I've got both of these now, thanks for the catch